### PR TITLE
Lrn it2 game stats

### PR DESCRIPTION
**Pull request by LancePants97**
Updated all my methods in game_stats to be working correctly. Also added some tests in game_stats_spec for the helper methods I created. I ran the spec harness on everything I wrote and it is all working as intended!

### CODE REVIEW

File: lib/game_stats.rb [hunk 1]
```diff
@@ -1,4 +1,70 @@
 
 
+
+
 module GameStats
+
+	def highest_total_score # integer
```
Comment (cheeleertr) [inline lib/game_stats.rb:7]: One line methods. Very clean.

File: data/games_fixture.csv
Comment (cheeleertr) [inline data/games_fixture.csv:1]: Good add to get better testing data

File: lib/stat_tracker.rb
Comment (cheeleertr) [inline lib/stat_tracker.rb:1]: Good update for spec_harness

File: spec/game_stats_spec.rb
Comment (cheeleertr) [inline spec/game_stats_spec.rb:1]: Good tests. Might want to make a test for the #all_scores helper method later.

File: lib/game_stats.rb [hunk 2]
```diff
@@ -1,4 +1,70 @@
...
+		all_scores.sort.pop
+	end
+
+	def lowest_total_score # integer
+		all_scores.sort.shift
+	end
```
Comment (jdmchugh111) [inline lib/game_stats.rb:13]: Really like the method chaining here. Could have maybe used a .last and .first (but they'll give you the same return value so it doesn't really matter)

File: lib/game_stats.rb [hunk 3]
```diff
@@ -1,4 +1,70 @@
...
+
+	def percentage_home_wins # float
+		total_games_float = @games.count.to_f
+		percentage = home_winners.to_f / total_games_float
+		percentage.round(2)
+	end
+
+	def percentage_visitor_wins # float
```
Comment (jdmchugh111) [inline lib/game_stats.rb:21]: agree with Chee, really clean methods. great job, Lance

**Review by jdmchugh111 — APPROVED** [review]

**Review by JCL461437 — COMMENTED** [review]
Looks great Lance!

**Pull request merged**